_posts/pl/2022-01-11-smarthome.md

```diff
@@ -0,0 +1,316 @@
+#### Przekaźniki oraz gniazdka
+
+Pozwoliłem sobie zebrać te wszystkie urządzenia w jedną grupę, bo ich głównym przeznaczeniem jest włączenie/wyłączenie zasilania. 
+Różnią się jedynie wspieranymi funkcjami. Do których można zaliczyć wybór typu klawisza (w przypadku przekaźników), 
+liczbę i rodzaj raportowanych danych. Wybrane informują o poborze energii, temperaturze, a także pozwalają ustawić stan po powrocie zasilania. 
+Warto zwrócić też uwagę na to, aby mieściły się w europejskich puszkach (jeśli są to przekaźniki).
+Tak jak wcześniej już wspomniałem, zaletą tych urządzeń jest to, że większość z nich wzmacnia zasięg sieci ZigBee.
+
+![Przekaźnik](/assets/img/posts/2022-01-11-smarthome/relay.png){: width="250" }
+![Gniazdko](/assets/img/posts/2022-01-11-smarthome/socket.png){: width="250" }
+
+
+#### Sterowniki taśm led
+
+Sterowniki taśm led, czyli urządzenia, które pozwalają nam na wybór koloru oraz jasności.
+W swoim SmartHome mam tylko jedno takie urządzenie, które działa prawie bez zastrzeżeń.
+Jedyną wadą jest to, że po powrocie zasilania jest domyślnie włączone i musiałem to „ograć” dodatkowym przekaźnikiem i automatyzacją.
+Z zalet — podłączamy dowolną taśmę LED oraz dowolny zasilacz (oczywiście dobieramy urządzenia w zależności od potrzebnej mocy i napięcia).
+W moim wypadku jest to taśma LED zasilona z 24V na dwóch końcach (dla równomiernego podświetlenia, w przypadku większych długości).
+
+![Sterownik taśmy LED](/assets/img/posts/2022-01-11-smarthome/ledcontroller.png){: width="250" }
+
+
+#### Przyciski
+
+W przypadku przycisków podobnie jak i w przypadku przekaźników jest w czym przebierać. 
+Szczególnie lubię te najmniejsze, z dwoma przyciskami pochodzące z rodziny IKEA Tradfri. 
+Są dosyć tanie, poza krótkim przyciśnięciem obsługują także te długie, więc możemy wykorzystać je nawet do ściemniania i rozjaśniania.
+W moim przypadku z ich pomocą dołożyłem obsługę głównego oświetlenia kuchennego w bardziej wygodnym miejscu — pod okap.
+Przyciski mają wbudowany magnes, więc nie było problemu z ich przymocowaniem w tym miejscu.
+Zanim odkryłem te przyciski, miałem kostkę Aqara Cube. Dosyć fajny gadżet z dużą liczbą zdarzeń.
+Przewrócenie o 90°, 180°, obrót ze wskazaniem kąta, podwójne uderzenie.
+Jednak jest dużo mniej intuicyjny od standardowych przycisków i trzeba wiedzieć, co kryje się pod każdą z akcji na kostce.
+
+![Przycisk IKEA Tradfri](/assets/img/posts/2022-01-11-smarthome/tradfriswitch1.png){: width="250" }
+![Przycisk IKEA Tradfri](/assets/img/posts/2022-01-11-smarthome/tradfriswitch2.png){: width="250" }
+![Kostka Aqara](/assets/img/posts/2022-01-11-smarthome/aqaracube.png){: width="250" }
+
+
+#### Czujniki ruchu
+
+W moich automatyzacjach używam jedynie czujników ruchu od Xiami/Aqara.
+Mogę o nich powiedzieć, że są dosyć małe, stąd łatwo wpasować je w estetykę pomieszczenia.
+Z wad to w domyślnej konfiguracji mogą się nie sprawdzić we wszystkich automatyzacjach, ponieważ raportują swój stan co około minutę-półtora.
```

Comment:
Mogą się jednak nie sprawdzić w domyślnej konfiguracji, zwłaszcza we wszystkich automatyzacjach, ponieważ raportują swój stan co około minutę-półtora.